intl/msg_hash_us.h

```diff
@@ -1561,6 +1561,14 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_VIDEO_FULLSCREEN_Y,
    "Set the custom height size for the non-windowed fullscreen mode. Leaving it unset will use the desktop resolution."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_VALUE_VIDEO_FORCE_RESOLUTION,
+   "Force resolution on UWP"
+   )
+MSG_HASH(
+   MENU_ENUM_SUBLABEL_VIDEO_FORCE_RESOLUTION,
+   "Force the resolution to the fullscreen size, if set to 0, a fixed value of 3840 x 2160 will be used."
```

Comment:
I'm sorry, what does "Force the resolution to the fullscreen size" even mean? The fullscreen of what?